src/pages/index.astro

```diff
@@ -11,14 +11,28 @@ import Chat from "../components/Chat.astro";
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width" />
-    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
     <meta name="generator" content={Astro.generator} />
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
+    <link rel="manifest" href="/site.webmanifest" />
+    <link rel="stylesheet" href="/src/styles/main.css" />
     <title>Luna</title>
   </head>
 
-  <body>
-    <div class="grid place-items-center h-screen content-center">
-      <Chat />
-    </div>
+  <body class="flex flex-col h-screen w-screen">
+    <nav class="flex items-center px-8 gap-2 h-16">
+      <h1 class="text-xl">LUNA</h1>
+      <img src="logo.png" class="h-8" />
+    </nav>
+    <main
+      class="flex flex-grow items-center justify-center h-[calc(100vh-04rem)]">
```

Comment:
This looks 💯! 
I wish TW had a better way to remove "space" from `100vh` but not yet.